subprojects/docs/src/samples/build-organization/standalone-precompiled-plugins/kotlin/src/main/kotlin/com.example.service.gradle.kts

```
@@ -0,0 +1,30 @@
+// tag::plugins[]
+plugins {
+    id("com.example.java-convention")
+}
+// end::plugins[]
+
+val integrationTest by the<SourceSetContainer>().creating {
```

:x: type safe accessor to the `SourceSetContainer` extension should be usable here
```suggestion
val integrationTest by sourceSets.creating {
```